benchmarks/nsinker_spherical_shell/nsinker_gmg.prm

```diff
@@ -0,0 +1,95 @@
+# This is the "Spherical Shell NSinker" benchmark
+
+set Additional shared libraries            = ./libnsinker.so
+
+set Dimension                              = 3
+set End time                               = 0
+set Pressure normalization                 = volume
+set Nonlinear solver scheme                = no Advection, single Stokes
+set Use years in output instead of seconds = false
+
+subsection Solver parameters
+  subsection Matrix Free
+    set Output details = true
+  end
+  subsection Stokes solver parameters
+    set Stokes solver type = block GMG
+    set Number of cheap Stokes solver steps             = 500
```

Comment:
This is different compared to the box case. Does it make sense to explain somewhere why for spherical models, it's useful to pick a larger value?

Reply:
For nsinker we followed parameters from the literature, while I don't have to do that here. GMG typically does not need expensive iterations. (I have not tested this here, yet)

Reply:
We've had some problems with the expensive iterations too, for several projects. I think everyone in my group who's using the GMG uses something like a few thousand cheap solver steps to make sure we never use the expensive solver. It's of course a completely separate issue from this PR, but maybe we can document somewhere that it makes sense to use a lot of cheap steps with the GMG, or change the default.